c10/xpu/XPUFunctions.cpp

```diff
@@ -65,7 +75,7 @@ inline void initDevicePoolCallOnce() {
   c10::call_once(init_flag, initGlobalDevicePoolState);
 }
 
-void initDeviceProperties(DeviceProp* device_prop, int device) {
+void initDeviceProperties(DeviceProp* device_prop, DeviceIndex device) {
```

Comment:
btw if you want to find all other raw ints to remove them, regex github search is pretty effective these days! https://github.com/search?q=repo%3Apytorch%2Fpytorch+%2F%28+%7C%2C%7C%5E%29int+%2F+language%3AC%2B%2B++path%3A%2F%5E%28c10%5C%2Fxpu%5C%2F%7Caten%5C%2Fsrc%5C%2FATen%5C%2Fxpu%5C%2F%29%2F&type=code

Reply:
@cyyever [[DeviceIndex][7/N] Use DeviceIndex in XPU](https://github.com/pytorch/pytorch/pull/120576#top) help us to replace other raw ints with DeviceIndex.